shimmingtoolbox/prepare_fieldmap.py

```diff
@@ -137,6 +162,8 @@ def correct_2pi_offset(unwrapped, mag, mask, validity_threshold):
 
     """
     # Create a mask that excludes the noise
+    # if len(np.shape(mag)) == 3:
+    #     mag = np.repeat(mag[..., np.newaxis], len(unwrapped), axis=3)
```

Comment:
Is this necessary?

Reply:
No, it's not. First I wanted to change the function itself but then decided not to do that.